test_runner/fixtures/pg_version.py

```diff
@@ -0,0 +1,57 @@
+import enum
+import os
+from typing import Iterator
+
+import pytest
+from _pytest.config.argparsing import Parser
+from pytest import FixtureRequest
+
+from fixtures.log_helper import log
+
+"""
+This fixture is used to determine which version of Postgres to use for tests.
+"""
+
+
+# Inherit PgVersion from str rather than int to make it easier to pass as a command-line argument
+# TODO: use enum.StrEnum for Python >= 3.11
+@enum.unique
+class PgVersion(str, enum.Enum):
+    V14 = "14"
+    V15 = "15"
+    # Instead of making version an optional parameter in methods, we can use this fake entry
+    # to explicitly rely on the default server version (could be different from pg_version fixture value)
+    NOT_SET = "<-POSTRGRES VERSION IS NOT SET->"
+
+    @staticmethod
+    def from_int(int_version: int) -> "PgVersion":
+        return PgVersion(str(int_version)[:2])
+
+    # Make it less confusing in logs
+    def __repr__(self) -> str:
+        return f"'{self.value}'"
```

Comment:
NIT: `f"PG{self.value}"`? WDYT?

Reply:
Then logs will be like this:
```
Run ['neon_local', '--pg-version', PG14, ... ]
```

This is what I meant by confusing.
So I'd stick to the current version